idl-meta/src/lib.rs

```diff
@@ -26,3 +26,30 @@ pub trait ServiceMeta {
     fn commands() -> MetaType;
     fn queries() -> MetaType;
 }
+
+pub struct AnyServiceMeta {
```

Comment:
I think I miss the point of introducing new type. Is `ServiceMeta` not object safe or something?

Reply:
The point of this struct is to represent arbitrary service meta in program meta. The latter can have multiple services hosted by a program, so they a represented via an iterator. The `ServiceMeta` trait can't be used for iterator item, so this struct has got introduced